app/javascript/components/BotLayout.jsx

```diff
@@ -25,30 +26,37 @@ export class BotLayoutComponent extends Component {
     const { botsLoaded, bot, children, botActions } = this.props
 
     if (botsLoaded == true && bot != null) {
-      return <AppLayout
-        title={
-          <EditableTitleLabel
-            title={bot.name}
+      return (
+        <AppLayout
+          title={
+            <EditableTitleLabel
+              title={bot.name}
             onSubmit={(name) => { botActions.updateBot({...bot, name}) }} />
-        }
-        headerNavLinks={[
-          // <HeaderNavLink label="Analytics" to="#" />,
-          // <HeaderNavLink label="Data" to="#" />,
-          // TODO: use active="/b/:id/channel"  to allow deep linking
-          <HeaderNavLink label="Channel" to={r.botChannel(bot.id)} />,
-          <HeaderNavLink label="Behaviour" to={r.botBehaviour(bot.id)} />,
-          // <HeaderNavLink label="Translations" to="#" />,
-          // <HeaderNavLink label="Collaborators" to="#" />,
-        ]}
-        buttonAction={() => botActions.publishBot(bot)} buttonIcon="publish"
+          }
+          headerNavLinks={[
+            // <HeaderNavLink label="Analytics" to="#" />,
+            // <HeaderNavLink label="Data" to="#" />,
+            // TODO: use active="/b/:id/channel"  to allow deep linking
+            <HeaderNavLink label="Channel" to={r.botChannel(bot.id)} />,
+            <HeaderNavLink label="Behaviour" to={r.botBehaviour(bot.id)} />,
+            <HeaderNavLink label="Translations" to={r.botTranslations(bot.id)} />,
+            // <HeaderNavLink label="Collaborators" to="#" />,
+          ]}
+          actionLinks={[
+            <HeaderNavLink label="Rename" />,
+            <HeaderNavLink label="Unpublish" onClick={() => botActions.unpublishBot(bot)}/>,
```

Comment:
Rather than HeaderNavLink I would use HeaderNavAction for onClick. Leaving NavLink as react-dom-router related.